_posts/2020-09-01-what-is-code-coverage-ruby-on-rails.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: post
+title: "What are the Code Coverage Metrics for Ruby on Rails?"
+date: 2020-09-01 10:00:00
+categories: ["rails"]
+authors: ["etagwerker"]
+---
+
+At FastRuby.io we are constantly looking at code coverage metrics for Ruby on
+Rails applications. It's a key indicator for us. We even use that information 
+to decide [whether we work on a Rails upgrade project or not](https://www.fastruby.io/blog/rails/upgrades/assessing-rails-upgrades.html).
+
+So, I was interested in seeing code coverage metrics for the [Ruby on Rails framework](https://github.com/rails/rails). 
+I couldn't find any information about this online, so I decided to generate a 
+few reports for each component.
+
+This is an article about my process and my findings.
+
+<!--more-->
+
+## Process
+
+In order to [calculate code coverage](https://www.fastruby.io/blog/rails/upgrades/simplecov/rails-upgrades-simplecov.html), 
+I used [SimpleCov](https://github.com/simplecov-ruby/simplecov) and 
+analyzed Rails at [f22dd39](https://github.com/rails/rails/commit/f22dd39cb2adf85d3deeca61f9465206f7bd8df3).
+
+<<<<<<< HEAD:_posts/2020-09-01-what-is-code-coverage-ruby-on-rails.md
```

Comment:
@etagwerker  This seems to be a git conflict, so I'm not sure which sentence to review. Could you fix them before I proceed?

Reply:
@cleicar Just addressed your comment. Good catch! 💯